test/ContractGenerator.Tests/ProtoUtilsTests.cs

```diff
@@ -67,6 +67,32 @@ public void GetClassName_ReturnsCorrectClassName()
         Assert.Equal("global::AElf.Contracts.HelloWorld.HelloWorld", className);
     }
 
+    [Fact]
+    public void GetCsharpComments_ReturnsComments()
+    {
+        // Arrange: Create a DescriptorBase with a known FullName and File
+        var fds = GetFileDescriptorSet("helloworld");
+        var byteStrings = fds.File.Select(f => f.ToByteString());
+        var fileDescriptors = FileDescriptor.BuildFromByteStrings(byteStrings, _extensionRegistry);
+        var file = fileDescriptors[^1];
+
+        // Act: Call the GetClassName method
+        var comments = ProtoUtils.GetCsharpComments(file, true);
+        const string expectedComments = @"// These are test header comments!
+// The namespace of this class
+// The name of the state class the smart contract is going to use to access blockchain state
+// Actions (methods that modify contract state)
+// Stores the value in contract state
+// Views (methods that don't modify contract state)
+// Get the value stored from contract state
+// An event that will be emitted from contract method call
```

Comment:
These are not file level comments. Is this output correct? It makes no sense to print out the comments for methods and messages like this.

Reply:
hmm these comments are present in the file tho i suppose it would be best to just print the top-level file-proto comments? i.e just `// These are test header comments!` based on the example below. wdyt?

```
syntax = "proto3";
// These are test header comments!
import "aelf/options.proto";
import "google/protobuf/empty.proto";
import "google/protobuf/wrappers.proto";
// The namespace of this class
option csharp_namespace = "AElf.Contracts.HelloWorld";

service HelloWorld {
    // The name of the state class the smart contract is going to use to access blockchain state
    option (aelf.csharp_state) = "AElf.Contracts.HelloWorld.HelloWorldState";

    // Actions (methods that modify contract state)
    // Stores the value in contract state
    rpc Update (google.protobuf.StringValue) returns (google.protobuf.Empty) {
    }

    // Views (methods that don't modify contract state)
    // Get the value stored from contract state
    rpc Read (google.protobuf.Empty) returns (google.protobuf.StringValue) {
        option (aelf.is_view) = true;
    }
}

// An event that will be emitted from contract method call
message UpdatedMessage {
    option (aelf.is_event) = true;
    string value = 1;
}

```